roadmap.md

```diff
@@ -31,15 +25,14 @@ state.
 
 ### Work in progress (WIP)
 
-1. **SLSA compliance in the Kubernetes Release Process (Secure)**
+1. **Make Artifact Validation More Robust**
 
-   Outcome: Ensure that our release process is [SLSA](https://slsa.dev)
-   compliant. We also intend to participate actively in the development of the
-   framework.
+   Outcome: Provide a fast and error-resistant workflow for artifact promotion
+   compliant with SLSA.
 
-   Enhancement: https://github.com/kubernetes/enhancements/issues/3027
+   Enhancement: was https://github.com/kubernetes/enhancements/issues/3027
 
-   Project board: https://github.com/orgs/kubernetes/projects/138
+   Project board: https://github.com/orgs/kubernetes/projects/171
 
 1. **Moving deb/rpm package builds to community infrastructure (Consumable)**
```

Comment:
This is in fact done, so I think we should rephrase this goal. I would split it into two deliverables:

- Making deb and rpm package builds more robust
  - The remaining work that we have is very well described with the provided robustness definition, e.g. writing tests, enabling reproducible builds...
  - Outcome can be similar to the outcome of artifact validation, e.g. provide a fast and error-resistant workflow for building and publishing packages to the community-owned infrastructure
- Enable other Kubernetes subprojects to use our packages infrastructure
  - This is about flexibility and it should cover efforts like building a CLI, library, writing documentation, etc. We don't have a KEP for this, but we can reuse the existing project board for now

Reply:
@xmudrii considering current resources, should we move both into the `Todo` section?

Reply:
Updated the roadmap to reflect the current state.